### PR TITLE
Use prettier

### PR DESCRIPTION
**Pull request by tptee**
This PR:

- adds `prettier` and `eslint-config-prettier`
- ran `prettier` on existing files
- runs `prettier` on staged files in a precommit hook

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,9 @@
----
-extends:
-	- formidable/configurations/es6-node
-rules:
-  func-style: off
-  arrow-parens: off
+{
```
Comment (ryan-roemer) [inline .eslintrc]: NIT: I prefer YAML and the alignment is off in JSON here. (And it was off in L3 in original yaml)

File: utils/format-assets.js [hunk 2]
```diff
@@ -7,41 +7,34 @@ function getAssets(stats) {
...
-      .filter(asset => asset.name.indexOf("hot-update") < 0)
-      .map(asset => {
-        const realBundleMatch = _.find({ path: asset.name })(bundles);
-        return realBundleMatch ? {
-          name: realBundleMatch.path,
-          size: realBundleMatch.metrics.meta.bundle.minGz,
-          minGz: true
-        } : asset;
-      })
+    assets.filter(asset => asset.name.indexOf("hot-update") < 0).map(asset => {
```
Comment (ryan-roemer) [inline utils/format-assets.js]: Hmmm... prettier newbie question -- any way to be able to do something like:

```js
assets
  .filter(
  .map(
````

etc? I like having each part of an array chain be newlined...

Comment (gartz) [reply utils/format-assets.js]: It’s prettier when not using prettier 😂

**Comment by ryan-roemer** [comment]
Too far behind master, so closing. We can pick up later with a fresh effort if we want!!!

**Pull request closed without merging**